File: beecrowd_ 1134.cpp
#include <stdio.h>
#include <string>

int main() {
  std::string ref[] = {"Alcool", "Gasolina", "Diesel"};
  int i, score[] = {0, 0, 0};
  while (scanf("%d", &i) == 1) {
    if (i != 4) {
      if (i < 4)
        ++score[i - 1];
    } else
      break;
  }
  printf("MUITO OBRIGADO\n");
  for (i = 0; i < 3; ++i) {
    printf("%s: %d\n", ref[i].c_str(), score[i]);
  }

  return 0;
}

File: beecrowd_1134.py
ref = ["Alcool","Gasolina", "Diesel"]
score = [0,0,0]
while True:
    i = int(input())
    if (i != 4):
        if (i < 4): 
            score[i - 1] += 1
    else: 
        break

print("MUITO OBRIGADO")
for i in range(3):
    print("%s: %d" %(ref[i],score[i]) )
